optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/drools/KieRuntimeBuilderWrapper.java

```diff
@@ -19,15 +19,19 @@
 import org.kie.api.KieBase;
 import org.kie.kogito.rules.KieRuntimeBuilder;
 
-public class KieBaseExtractor {
+/**
+ * Wraps {@link KieRuntimeBuilder} so the dependency on kogito-api is optional.
+ */
+public class KieRuntimeBuilderWrapper {
```

Comment:
Motivation for rename: We would have used KieRuntimeBuilder directly if it wasn't for the optional dependency requirement, so we should stay as close to that as possible. Other option as KieRuntimeBuilderProxy